### DISCUSSION TITLE
Possible to display numbers in a different font?

### DISCUSSION THREAD

**Discussion by Strykar** [kovidgoyal/kitty#4835] [Q&A] [answered]
If I have font family set to say Fira Code, can I display numbers in a different font?

**Comment by page-down** [comment, marked as answer]
You can configure `symbol_map` to use a different font for numbers.

https://sw.kovidgoyal.net/kitty/conf/#opt-kitty.symbol_map

**Comment by Strykar** [comment]
Thanks, and is it possible to set `adjust_line_height 5` for just one type of font, say, `bold_font`?

**Comment by page-down** [comment]
All cells are using the same width and height, so it is not possible to set a specific height for bold.
So if you use different fonts, then you need to make sure that the sizes of these fonts are compatible, otherwise you need to adjust them by the largest.
